kickstart/etc/smb.conf

```diff
@@ -39,3 +39,30 @@
    create mask = 0777
    directory mask = 0777
    writable = yes
+
+[omk]
+   comment = OpenMapKit Backups
+   path = /opt/data/backups/omk
+   browseable = yes
+   guest ok = yes
+   create mask = 0777
+   directory mask = 0777
+   writable = yes
```

Comment:
Drop these lines from each of the shares since we can inherit filesystem permissions:

```
   create mask = 0777
   directory mask = 0777
   writable = yes
```